gnmi_server/interface_description_test.go

```diff
@@ -64,11 +71,25 @@ func TestGetShowInterfaceDescription(t *testing.T) {
 			wantRetCode: codes.OK,
 			wantRespVal: []byte(expectedRetValue),
 			valTest:     false,
```

Comment:
If we set `valTest` false here, can we remove wantRespVal? As this is only an empty value.